wifiphisher/pywifiphisher.py

```diff
@@ -187,7 +187,8 @@ def shutdown(template=None, network_manager=None):
         template.remove_extra_files()
 
     print '[' + R + '!' + W + '] Closing'
-    sys.exit(0)
+    for c in phishinghttp.creds:
+        sys.exit('[' + R + '+' + W + '] Captured credentials:' + R + c)
```

Comment:
This doesn't look correct. If `phishinghttp.creds` list contains more than one element, it will print only one pair of creds and exit on the first iteration.